sutera-lib/src/signature/message.rs

```diff
@@ -22,15 +24,18 @@ pub enum SuteraMessageSigningError {
 
 impl SuteraSignedMessage {
     /// Sign a message with the author's signing key.
+    /// 署名者の署名鍵でメッセージを署名する。
```

Comment:
```suggestion
    /// Sign a message with the [`author`]'s signing key.
    /// [`author`]の署名鍵でメッセージを署名する。
```
構造体内の具体的なプロパティ名を指してるやつは訳さないでOK！
(今回はもともとの英文がプロパティを参照する表記になってなかったので my fault ですごめんね……一緒に直しておいてくれるとありがたい)